slimline.zsh

```diff
@@ -106,6 +112,11 @@ prompt_slimline_set_rprompt() {
   if [[ -n "${_prompt_slimline_git_output:-}" ]]; then
     RPROMPT+="${RPROMPT:+ }${_prompt_slimline_git_output}"
   fi
+
+  # add virtualenv (if active)
+  if (( ${SLIMLINE_DISPLAY_VIRTUALENV:-1} )); then
+    RPROMPT+="${RPROMPT:+ }$(prompt_slimline_virtualenv)"
```

Comment:
This adds a trailing space to the prompt if `VIRTUAL_ENV ` is empty. We can store it in a variable first and add it only if it is not empty:
```shell
local virtual_env="$(prompt_slimline_virtualenv)"
if [[ -n "${virtual_env}" ]]; then
  RPROMPT+="${RPROMPT:+ }${virtual_env}"
fi
```